example/AsProject/Physical/ARM/Hardware.hw

```diff
@@ -8,7 +8,6 @@
       <Parameter ID="ActivateDevice" Value="1" />
     </Connector>
     <Parameter ID="ConfigurationID" Value="AsProject_ARM" />
-    <Parameter ID="Simulation" Value="1" />
```

Comment:
Should this be changed?

Reply:
I was always under the assumption (maybe incorrectly) that we don't push the 'Simulation Active' setting into repos.  Is this a correct assumption?